models/pokemon_model.go

```diff
@@ -0,0 +1,6 @@
+package models
+
+type Pokemon struct {
+	ID   int
```

Comment:
I'm not very familiar with how Gin works, but you would probably still want to define the json annotations here in your model, for example 

```
ID int `json:"id"`
```

And so on for the rest of your fields